package.json

```
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
     "start": "NODE_PATH=./build node build/index.js",
     "build": "tsc -p .",
     "dev": "nodemon src/index.ts",
```

Como o arquivo index foi separado em app  e server, mudar aqui para:
```suggestion
    "dev": "nodemon src/server.ts",
```